10.21.0/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:10.21.0-alpine3.11
```

Comment:
I would use `10-alpine` as a base so as to avoid pinning ourselves to an older version

Reply:
Changed 👍